app/uk/gov/hmrc/transitmovementseisstub/connectors/EISConnector.scala

```diff
@@ -41,18 +48,30 @@ trait EISConnector {
 class EISConnectorImpl(
   val code: String,
   val eisInstanceConfig: EISInstanceConfig,
-  httpClientV2: HttpClientV2
+  httpClientV2: HttpClientV2,
+  clock: Clock
 )(implicit
   ec: ExecutionContext,
   val materializer: Materializer
 ) extends EISConnector
     with Logging {
 
+  private val HTTP_DATE_FORMATTER = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss", Locale.ENGLISH).withZone(ZoneOffset.UTC)
+
+  private def nowFormatted(): String =
+    s"${HTTP_DATE_FORMATTER.format(OffsetDateTime.now(clock.withZone(ZoneOffset.UTC)))} UTC"
+
   override def post(body: Source[ByteString, _])(implicit hc: HeaderCarrier): Future[Either[RoutingError, Unit]] =
     httpClientV2
       .post(url"${eisInstanceConfig.url}")
       .setHeader(
-        hc.headers(Seq("X-Conversation-Id", "X-Correlation-Id", "Content-Type", "Accept", "Date")): _*
+        hc.headers(Seq("X-Conversation-Id", "X-Correlation-Id")): _*
+      )
+      .setHeader(
+        HeaderNames.AUTHORIZATION -> hc.authorization.get.value,
```

Comment:
Possibility of getting exception at this point if authorisation is not present in header carrier?

Reply:
If it's not there we have bigger problems - it must always be there.